pages/api/image.ts

```diff
@@ -0,0 +1,58 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import { IncomingForm, PersistentFile } from "formidable";
+import absoluteUrl from 'next-absolute-url'
+
+const IMAGE_BASE_URL = 'https://imageing.org/shared/';
+
+declare module 'formidable' {
+    interface FileJSON {
+        newFilename: string;
+    }
+}
+
+export default function handler(req: NextApiRequest, res: NextApiResponse) {
+    if (req.method === "POST") {
+        const form = new IncomingForm({

```

Comment:
Have to think about a different way on handling those images.
I cant save them to the server with Netlify/Vercel.